src/yoroi-wallets/cardano/byron/ByronWallet.ts

```diff
@@ -104,8 +107,11 @@ export type ByronWalletJSON = Omit<ShelleyWalletJSON, 'account'>
 
 export type WalletJSON = ShelleyWalletJSON | ByronWalletJSON
```

Comment:
maybe it shouldn't be here

Reply:
I want to change Byron Wallet as little as possible, and delete as soon as possible